packages/rx_bloc_cli/mason_templates/bricks/lib_pin_code/__brick__/lib/lib_pin_code/views/create_pin_page.dart

```diff
@@ -0,0 +1,130 @@
+{{> licence.dart }}
+
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:widget_toolkit/models.dart';
+import 'package:widget_toolkit/ui_components.dart';
+import 'package:widget_toolkit_biometrics/widget_toolkit_biometrics.dart';
+import 'package:widget_toolkit_pin/widget_toolkit_pin.dart';
+
+import '../../app_extensions.dart';
+import '../../lib_router/blocs/router_bloc.dart';
+import '../../lib_router/router.dart';
+import '../bloc/create_pin_bloc.dart';
+import '../models/pin_code_arguments.dart';
+import '../services/create_pin_code_service.dart';
+
+class CreatePinPage extends StatefulWidget {
```

Comment:
- Currently the page does not cover the navigation items. Please update it, so it's pushed above them. 
- If during the "Create Pin" flow the "Confirm pin code" is mistaken, it does not allow you to enter a new one. 
- When the pin creation flow is completed, the screen remains visible and the user can press the biometrics button, which in turn shows an error message.

Reply:
The first bullet is done